interface/interface.dm

```diff
@@ -60,7 +60,8 @@
 			message += GLOB.revdata.GetTestMergeInfo(FALSE)
 		if(tgalert(src, message, "Report Issue","Yes","No")=="No")
 			return
-		src << link("[config.githuburl]/issues/new")
+		var/static/issue_template = file2text(".github/ISSUE_TEMPLATE.md")
+		src << link("[config.githuburl]/issues/new[GLOB.round_id ? "?body=Issue reported from Round ID: [GLOB.round_id]\n\n[issue_template]" : ""]")
 	else

```

Comment:
Would it be better to encapsulate the body parameter's value with url_encode?

Reply:
proabably, because the full ISSUE_TEMPLATE.md isn't going through atm